src/newmagic.cpp

```diff
@@ -1663,13 +1663,15 @@ void cast_health_spell(struct char_data *ch, int spell, int sub, int force, char
       }
 
       base_target = 0;
+      bool drugged = FALSE;
       for (int i = MIN_DRUG; i < NUM_DRUGS; i++) {
         if (GET_DRUG_STAGE(vict, i) != DRUG_STAGE_UNAFFECTED) {
           base_target = MAX(base_target, drug_types[i].power);
```

Comment:
Since we have some drugs with power 0, we probably need to rethink this portion of it, otherwise the spell is working against no TN. Does the book say anything about how this should be handled?

Reply:
I haven't been able to find any mention of a power rating for non-damaging drugs, though I only know of rules involving drugs in SR3 and M&M. I'm thinking this is a question for Vile.